app/src/gui/components/record/relationships/RelatedInformation.tsx

```diff
@@ -675,15 +698,9 @@ function get_field_label(ui_specification: ProjectUIModel, field: string) {
     return {field_name, is_deleted};
   }
   try {
-    if (
-      ui_specification['fields'][field]['component-parameters'][
-        'InputLabelProps'
-      ]['label']
-    )
+    if (ui_specification['fields'][field]['component-parameters'].label)
       field_name =
-        ui_specification['fields'][field]['component-parameters'][
```

Comment:
What's the rationale for this change? I notice you've removed InputLabelProps?

Reply:
`InputLabelProps` was an old way to set a label on this and a few other fields, I've now standardised to `['component-parameters'].label` for all field types.